src/Microsoft.ML.Data/Scorers/ScoreMapperSchema.cs

```diff
@@ -4,327 +4,106 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.ML.Internal.Utilities;
 
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// A base class for schemas for ISchemaBoundMappers. Takes care of all the metadata that has to do with
-    /// the score column. If the predictor schema has more than one output column, then the GetColumnType(),
-    /// TryGetColumnIndex() and GetColumnName() methods should be overridden. If additional metadata is
-    /// needed, the metadata methods can also be overridden.
+    /// This class contains method for creating commonly used <see cref="Schema"/>s.
     /// </summary>
-    public abstract class ScoreMapperSchemaBase : ISchema
+    [BestFriend]
+    internal static class ScoreSchemaFactory
```

Comment:
Can we rename this file to match the class that is inside of it?  `ScoreSchemaFactory.cs`

Reply:
Sure.